src/frontend/components/UI/Header/index.css

```diff
@@ -18,15 +18,6 @@
   );
 }
 
-@media screen and (max-width: 1000px) {
-  .Header {
-    grid-template-columns: 1fr min-content;
-    grid-template-areas:
-      'search search'
-      'filters filters';
```

Comment:
we don't need to drop into a second row now that the `filters` element is smaller